internal/commands/root/http.go

```diff
@@ -47,7 +50,7 @@ var AcceptedCiphers = []uint16{
 	tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
 }
 
-func loadTLSConfig(certPath, keyPath, caPath string, allowUnauthenticatedClients bool) (*tls.Config, error) {
+func loadTLSConfig(ctx context.Context, certPath, keyPath, caPath string, allowUnauthenticatedClients bool) (*tls.Config, error) {
```

Comment:
Line 58: we should use `clientAuth = tls.RequestClientCert`
refer to https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/server.go#L1051